validator/sawtooth_validator/server/core.py

```diff
@@ -294,15 +294,13 @@ def __init__(self,
             permission_verifier=permission_verifier)
 
         chain_controller = ChainController(
-            block_cache=block_cache,
-            block_validator=block_validator,
-            state_view_factory=state_view_factory,
-            chain_head_lock=block_publisher.chain_head_lock,
-            on_chain_updated=block_publisher.on_chain_updated,
-            chain_id_manager=chain_id_manager,
-            data_dir=data_dir,
-            config_dir=config_dir,
-            chain_observers=[
+            block_store,
+            block_cache,
+            block_validator,
+            block_publisher.chain_head_lock,
+            block_publisher.on_chain_updated,
+            data_dir,
```

Comment:
I believe kwargs were used previously to mitigate ordering mistakes made when the constructor updates. Why did you decide to remove them?